README.md

```diff
@@ -45,3 +45,11 @@ rsync -arv output/ user@target:/path/to/output/
 ```
 
 - For Apache, allow the deployed `.htaccess` to override settings for the active `Directory`
+
+## View
+
+You can view the output using Docker by running the following:
+
+```
+docker run -dit --rm --name irods.org -p 9090:80 -v $(pwd)/output:/usr/local/apache2/htdocs/ httpd:2.4
```

Comment:
it's either `-d` or `-it`, right?

not both?

Reply:
I just copied that from the irods_docs repo. I think `-d` is all that's needed.

Let me confirm that real quick.

Reply:
Done.